docs/extras/transfer_sd.md

```diff
@@ -1,49 +1,49 @@
 
 ## Instructions
 
 You should first check whether you have a file or partition based emuMMC:
-    
+
 1.  Inject the hekate payload.
 2.  Tap on `emuMMC`.
 3.  Under `emuMMC Info & Selection`, check the text next to `Type`.
     - If you have one it should be either `SD Raw Partition` or `SD File`.
 
 -----
-### If you are using a file based emummc or are using no emummc at all:
-        
-1.  Remove your SD card.   
-2.  Insert your old SD card into your computer.
-3.  Copy the contents of your SD card somewhere to your PC.
-4.  Eject the SD card and insert your new one.
-5.  Format your new SD card to FAT32 if it isn’t already.
+### If you are using a file based emuMMC or are using no emuMMC at all:
+
+1.  Remove your microSD Card.   
+2.  Insert your old microSD Card into your computer.
+3.  Copy the contents of your microSD Card somewhere to your PC.
+4.  Eject the microSD Card and insert your new one.
+5.  Format your new microSD Card to FAT32 if it isn’t already.
     - To do this, use [guiformat](http://ridgecrop.co.uk/index.htm?guiformat.htm) for example (Windows).
-6.  Copy the files from your PC to your new SD card and you’re done.
+6.  Copy the files from your PC to your new microSD Card and you’re done.
 
 -----
-### If you are using a partition based emummc:
-    
+### If you are using a partition based emuMMC:
+
 !!!warning "Space for the backup"
     You need at least 30GB of free space to be able to backup and restore the emuMMC!
 
 1.  Inject the hekate payload.
 2.  In the main menu, tap on `Tools`, then `Backup eMMC` and set `SD emuMMC Raw Partition` at the bottom of your screen to `ON`.
 3.  Backup both `SD emuMMC BOOT0 & BOOT1` and `SD emuMMC RAW GPP` (Note: raw gpp may take a while).
-4.  Once both are done, go back to the main menu, remove your SD card and insert it into your PC.
+4.  Once both are done, go back to the main menu, remove your microSD Card and insert it into your PC.
 5.  If Windows asks you to format a drive, discard it and select the drive with your SD contents.
-6.  Copy the contents of your SD card somewhere to your PC.
-7.  Follow only [this page](https://switchgui.de/switch-guide/user_guide/emummc/partitioning_sd/) to partition the new SD card for an emuMMC setup.
-8.  After it’s done, exit and insert your new SD card into your PC.
-9.  Copy the contents of your old SD card to your new one.
-10. Navigate to `/backup/<some characters>/` on your SD card and move `BOOT0`, `BOOT1` and the `rawnand.bin.xx` files to the `restore` folder.
-11. Eject the SD card and insert it into your switch.
-12. Inject the hekate payload once again.
+6.  Copy the contents of your microSD Card somewhere to your PC.
+7.  Follow only [this page](https://switchgui.de/switch-guide/user_guide/emummc/partitioning_sd/) to partition the new microSD Card for an emuMMC setup.
```

Comment:
Might as well change the link from "https://switchgui.de/switch-guide/user_guide/emummc/partitioning_sd/" to "../user_guide/emummc/partitioning_sd.md"